runtime/isp_run_app.py

```diff
@@ -210,39 +211,39 @@ def main():
         policies = ["none"]
 
     # use exiting policy directory if -p arg refers to path
-    if (len(policies) == 1 and 
-        "/" in args.policies[0] and
-        os.path.isdir(policies[0])):
-        policy_dir = os.path.abspath(policies[0])
+    if (len(policies) == 1 and  "/" in args.policies[0] and os.path.isdir(policies[0])):
         policy_name = os.path.basename(policy_dir)
```

Comment:
Has the `-p` flag with a pre-built policy path been tested with this change? It looks like it would fail.

Reply:
(Though I also have no idea when that feature of isp_run_app is ever actually used)

Reply:
Good question.  How do I pre-build a policy?

Reply:
Never mind, I figured it out.  The latest batch of commits should fix this (although the actual change this thread is addressing doesn't do anything but move the `if` condition to a single line because it's short).